examples/bri-3/docs/dids/did-authentication.md

```diff
@@ -0,0 +1,42 @@
+# Bri-3 DID Auth
+
+## DIDs Explanation
+
+[W3C Decentralized Identifiers](https://www.w3.org/TR/did-core/) (DIDs) are cryptographically variable, globally unique identifiers. By utilizing DIDs for authentication, they enable a decentralized and self-sovereign approach to digital identity, where users have control over their own identity data and can prove their identity without relying on centralized authorities.
+
+In context of Bri-3, they are designed to enable users (BpiSubjects) to
+
+1. Register/generate their own unique identifiers
+2. Prove their control over them by authenticating using digital signatures
+3. Obtain an access token as a proof of their authenticated identity that can be reused over time
+
+So that no sensitive information would be revealed during the business activities.
+
+## Variables Explanation
+
+**GOERLI_RPC_URL**="" # Any GOERLI RPC url i.e. "https://rpc.ankr.com/eth_goerli". This is used to resolve dids
```

Comment:
Please format this section so its easier to read. Suggestion: Italicise the example string and put the description on the next line.